import-export-cli/cmd/k8s/genDeploymentDir.go

```diff
@@ -163,10 +167,11 @@ func executeGenDeploymentDirCmd() error {
 			if errUnmarshal != nil {
 				utils.HandleErrorAndExit("Error unmarshal api configmap into struct ", errUnmarshal)
 			}
-			apiCrd.Name = apiMetaData.Name
-			apiCrd.Spec.SwaggerConfigMapName = fmt.Sprintf("%v-cm", apiMetaData.Name)
-			apiCrd.Spec.ParamsValues = fmt.Sprintf("%v-params", apiMetaData.Name)
-			apiCrd.Spec.CertsValues = fmt.Sprintf("%v-certs", apiMetaData.Name)
+			apiName := utils2.GetValidK8sResourceName(apiMetaData.Name)
```

Comment:
There is **fmt.Println(fileName)** in line 161 of this file. Shall we remove it, if it a debug log?

Reply:
Yes, Shall we remove that line?

Reply:
fixed with https://github.com/wso2/product-apim-tooling/pull/703/commits/b01140f8cb4f3c34eca58b05da64c057839cfddd